sandbox.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" xmlns="http://www.w3.org/1999/html">
```

Comment:
Why this ancient NS?

Reply:
No idea how that got in there, I definitely did not type it. Maybe IntelliJ generated it. Anyway, it's gone.